bindata/network/multus/002-rbac.yaml

```diff
@@ -97,3 +97,10 @@ rules:
   - update
   - patch
   - delete
+- apiGroups:
```

Comment:
Just to confirm, this is a ClusterRole - you want access to the whole cluster?

Reply:
Great eye! Thanks. Upstream, the namespace where these live is configurable, so it's convenient to have it that way upstream, but... We pin the namespace for leases in openshift, so we can use a role for this binding.

Really appreciate that. New commit incoming with a `Role` :+1:

Reply:
Also, fwiw... we DO want a `clusterrole` for the other permissions, as they can live in differing namespaces.